packages/frontend/src/components/SingleVideoComponents/VideoHeaderComponent.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react'
+import styled from 'styled-components';
+import { font } from '../../utils/font'
+import { image } from '../../utils/image'
+import { colors } from '../../utils/colors';
+
+const Header = styled.header`
+        display: flex;
+        justify-content: space-between;
+        padding-bottom: 20px;
+`;
+
+const Title = styled.h1`
+        font: ${font.HeaderFontMobile};
```

Comment:
rozbij to na FontSize, FontWeigt FontFamily itp. każde do innego enum

Reply:
rozbite na FontSize, FontWeigt FontFamily